### DISCUSSION TITLE
Fractional shares

### DISCUSSION THREAD

**Discussion by almanza-d** [kernc/backtesting.py#406] [Q&A] [answered]
https://github.com/kernc/backtesting.py/blob/0a76e9655f702331c4cc2a035d040d50111ce78d/backtesting/backtesting.py#L1070

My query is regarding the line above with fractional shares. Mainly I encounter this warning with some Bitcoin backtesting I have done recently. I am well aware that you can bypass this by simply increasing the initial cash, but I wanted to stay as true as possible to the initial cash. I personally tried to bypass this by manually setting the size to >1 on instances where equity did not allow for one whole unit. Unfortunately this does not execute properly and backtesting just ends after no more equity is left for future higher prices. 

Any insight on where the fractional shares will be available?

Or more directly, how can I go around this issue?

**Comment by kernc** [comment, marked as answer]
See https://github.com/kernc/backtesting.py/issues/134.
